offline_imu_cam/plugin.cpp

```diff
@@ -57,10 +57,9 @@ class offline_imu_cam : public ILLIXR::threadloop {
 	    RAC_ERRNO_MSG("offline_imu_cam at start of _p_one_iteration");
 		assert(_m_sensor_data_it != _m_sensor_data.end());
 #ifndef NDEBUG
-        std::chrono::time_point<std::chrono::nanoseconds> tp_dataset_now{std::chrono::nanoseconds{dataset_now}};
+		std::chrono::time_point<std::chrono::nanoseconds> tp_dataset_now{std::chrono::nanoseconds{dataset_now}};
```

Comment:
Should this be `std::chrono::time_point` or RelativeClock's `time_point`?

Reply:
1. I don't really care because it's just used in human readable output.
2. `dataset_now` is a dataset time, where 0 means 1970.